README.md

```diff
@@ -147,14 +147,16 @@ This command will retrieve the given package manager from the specified archive
 
 ## Contributing
```

Comment:
Perhaps this entire section should be moved to the CONTRIBUTING.md file? Then the README.md can link to it?